src/ipx.ts

```diff
@@ -66,6 +67,26 @@ export const ipxSetup: IPXSetupT = setupOptions => (providerOptions, moduleOptio
   }
 }
 
+async function getDevDirs (nuxt: Nuxt, moduleOptions: Parameters<ProviderSetup>[1]) {
+  let fs: { existsSync: (path: string) => boolean }
+  try {
+    // IPX is available in node context only
+    // Therefore it should be safe to require fs
+    fs = await import('node:fs')
```

Comment:
This code lives in builder namespace. You can safely import fs in top level of module